src/applications/vaos/appointment-list/redux/reducer.js

```diff
@@ -257,6 +257,7 @@ export default function appointmentsReducer(state = initialState, action) {
         confirmed,
         pending,
         appointmentDetails,
+        pendingStatus: FETCH_STATUS.notStarted,
```

Comment:
Did you look into why the logic above to modify the cancelled request in the `pending` list wasn't working?

Reply:
Yes, those are working as expected and the appointment details page changes to a canceled status just fine. The issue is when you navigate back to the requested list the fetching of pending appointments isn't triggered so you are looking at stale data.

My fix triggers a refresh only when the cancelation is successful and the user returns the the requested list.

Reply:
Right, I mean that the `pending` list in the Redux state is the state for the list page. The reducer is attempting to update that as well as the `appointmentDetails` state that the details page uses. It seems like that's not working, since the intent was that the list page would get updated after the change on the details page.

Reply:
Ok, that makes sense, I see what you are referring too.